### ISSUE TITLE
Raspberry pi 4 hotspot 2.4GHz, not 5Ghz.

### ISSUE THREAD

**Issue by egisz** [htruong/crankshaft#439]
I noticed that while in hotspot mode, crankshaft uses 2.4GHz mode, not 5GHz as it is recommended for better stability and quality.
I digged a bit and managed to fix it editing file `/etc/hostapd/hostapd.conf`: 
Changed lines: 
```
hw_mode=a
channel=40
```
It would be nice to autodetect that CS is running on RPI 3B+/4 and set these settings during install.
I remember we have revision detection in QT5 build scripts.
But it's also possible to check supported frequencies with command: 
`iw list`

**Comment by matt2005** [comment]
Yeah that's sounds good, I'll have a look at making a 2.4/5 option in gui and disable 5 from being selected if not supported. Just waiting for pi4 to arrive.
